RecoBTag/SecondaryVertex/python/candidateBoostedDoubleSecondaryVertexAK8Computer_cfi.py

```diff
@@ -8,3 +8,6 @@
     useGBRForest = cms.bool(True),
     useAdaBoost = cms.bool(False)
 )
+
+from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
+phase1Pixel.toModify(candidateBoostedDoubleSecondaryVertexAK8Computer, weightFile = cms.FileInPath('RecoBTag/SecondaryVertex/data/BoostedDoubleSV_AK8_BDT_PhaseI_v1.weights.xml.gz'))
```

Comment:
At the moment there is no plan. From the past experience it is much easier, more convenient, and faster to handle training files as externals. Perhaps things are better now with GTs but I doubt we can get a new GT created on the same times scale on which externals can be integrated in the IBs.